### PR TITLE
Change the language to option + 1, 2, 3

### PR DESCRIPTION
**Pull request by qlife1146**
The combination of option keys and 1, 2, and 3 allows you to change the language.
Please let me know if there is a problem.

### CODE REVIEW

File: src/json/switchLang to kor, eng, ja.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+	"title": "switch Language",
+	"rules": [
+		{
+			"description": "Use option+1/2/3/4 to switch language.",
```
Comment (MuhammedZakir) [inline src/json/switchLang to kor, eng, ja.json:5]: There is no mapping for `4`.

**Comment by MuhammedZakir** [comment]
Please use `_` or `-` instead of space in filename.

**Comment by qlife1146** [comment]
I'm sorry, I thought I could use the emoji keyboard, but I didn't modify it.

File: src/json/switchLanguage_to_kor_eng_ja.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{
+	"title": "switch_Language",
```
Comment (MuhammedZakir) [inline src/json/switchLanguage_to_kor_eng_ja.json:2]: For title, it is okay to use space. Rather, I recommend it to make the title easier to understand.

**Comment by MuhammedZakir** [comment]
This is optional: mention your config in `public/groups.json`[1].

[1] https://github.com/pqrs-org/KE-complex_modifications/blob/main/public/groups.json#L661

**Comment by qlife1146** [comment]
I modified title but is it okay?

**Comment by MuhammedZakir** [comment]
> I modified title but is it okay?

Yes.

**Comment by MuhammedZakir** [comment]
> This is optional: mention your config in `public/groups.json`[1].
> 
> [1] https://github.com/pqrs-org/KE-complex_modifications/blob/main/public/groups.json#L661

Will you be doing this? Or shall I merge this now?

**Comment by qlife1146** [comment]
merge now, please.

**Comment by qlife1146** [comment]
Thank you and have a nice day

**Comment by qlife1146** [comment]
How do I apply it to karabiner-elements? I can't find what I made in 'import more rules from the internet' by complex modificatoins.

**Comment by MuhammedZakir** [comment]
> How do I apply it to karabiner-elements? I can't find what I made in 'import more rules from the internet' by complex modificatoins.

There was a mistake. You need to put it in `public/json` for it to be shown in the website. I have fixed that in a new commit. The site should show it soon.

**Comment by qlife1146** [comment]
I solved it. Thank you.

**Comment by qlife1146** [comment]
greetings, The functions I made usually work well, and option+1 (Korean) is properly written in Korean on the top menu bar, but it is not written in Korean but in English.

**Comment by MuhammedZakir** [comment]
> greetings, The functions I made usually work well, and option+1 (Korean) is properly written in Korean on the top menu bar, but it is not written in Korean but in English.

Why only using `language`, the keyboard layout that is selected may not be what you expect. If that's the case, be more specific by using `input_source_id` and/or `input_mode_id`. Go to the end of below link to know how to find these for the keyboard layout you want.

ref: https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/conditions/input-source/

**Pull request merged**